frontend/src/static/js/v_authorship.js

```diff
@@ -26,7 +26,9 @@ const repoCache = [];
 const minimatch = require('minimatch');
 
 window.vAuthorship = {
-  props: ['info'],
+  props: {
+    info: Object,
```

Comment:
One question: won't declaring it in the previous way using `props: ['info']` will achieve the same purpose? Since you are watching the `info` prop.

Reply:
It will, but the way declaring it clearly is what Vue suggested though. 
https://vuejs.org/v2/guide/components-props.html#Prop-Types

Reply:
I believe declaring it clearly has better code quality.

Reply:
ok noted.